mage_ai/data_cleaner/transformer_actions/base.py

```diff
@@ -76,9 +76,9 @@ def axis(self):
     def execute(self, df, **kwargs):
         action_type = self.action['action_type']
         dependency = DEPENDENCIES.get(action_type, default_resolution)
-        dependencies_met, msg = dependency(df)
-        if not dependencies_met:
-            raise RuntimeError(f'Dependencies of this cleaning action are not completed: {msg}')
+        dependencies_met, msg = dependency(df, self.action)
```

Comment:
Shall we make DEPENDENCIES an empty dictionary?

Reply:
Yeah, I can make it empty and uncomment the error message so in the future the dependencies will work correctly.